lib/generators/access_lint/install/install_generator.rb

```diff
@@ -0,0 +1,29 @@
+module AccessLint
+  module Generators
+    class InstallGenerator < ::Rails::Generators::Base
+      source_root File.expand_path('../templates', __FILE__)
+
+      desc "Add AccessLint routes and include statements."
+      def insert_route
+        insert_into_file "config/routes.rb", :after => "Rails.application.routes.draw do\n" do
+          <<-RUBY
+  if Rails.env.test? || Rails.env.development?
+    mount AccessLint::Rails::Engine, at: "access_lint"
+  end
+          RUBY
+        end
+      end
+
+      desc "Add AccessLint include helper to application layout in test and development."
+      def insert_include_helper
+        insert_into_file "app/views/layouts/application.html.erb", :before => "</body>" do
+          <<-ERB
+    <% if Rails.env.test? || Rails.env.development? %>
+      <%= include_access_lint %>
+    <% end %>
+          ERB
```

Comment:
Another option might be to put this markup in a fixture-like file that you could then properly indent in the file and source from here without the heredoc at all.